mosdef_cassandra/writers/inp_functions.py

```diff
@@ -31,7 +31,7 @@ def generate_input(
     run_length : float
         simulation length in units of (default=steps), or your choice
         as specified by the 'units' option in **kwargs
-    temperature : unyt array
+    temperature : unyt_array
```

Comment:
Would it be more clear to call these `unyt_quantity`; I know a `unyt_quantity` is a `unyt array`. Just wondering if the terminology is confusing.

Reply:
I changed these to say `unyt_array or unyt_quantity` just to be clear.  From my understanding, these are pretty interchangeable.  For example, `isinstance(1 * u.nm, u.unyt_array` and `isinstance(1 * u.nm, u.array.unyt_quantity)` both return True